lib/Page.js

```diff
@@ -44,8 +47,6 @@ class Page extends EventEmitter {
     ]);
     if (ignoreHTTPSErrors)
       await client.send('Security.setOverrideCertificateErrors', {override: true});
-    const page = new Page(client, ignoreHTTPSErrors, screenshotTaskQueue);
```

Comment:
why this change?

Reply:
The FrameManager needs to be listening for execution contexts before Runtime.enable is called.